packages/web-app/public/index.html

```diff
@@ -47,6 +41,137 @@
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
+    <div class="loader">
+      <div class="logo">
+        <svg
+          id="eEqkoNHJ67b1"
+          xmlns="http://www.w3.org/2000/svg"
+          xmlns:xlink="http://www.w3.org/1999/xlink"
+          viewBox="0 0 195.547 126.359"
+          shape-rendering="geometricPrecision"
+          text-rendering="geometricPrecision"
+        >
+          <path
+            id="point"
+            d="M107.375,84.331c-12.283,5.413-26.535-.37-31.834-12.918-5.3-12.547.363-27.107,12.646-32.52c12.284-5.413,26.536.371,31.835,12.918c5.232,12.391-.22,26.78-12.257,32.343"
+            transform="translate(-.654591 2.954649)"
+            fill="#9a140c"
+            fill-rule="evenodd"
+          />
+          <path d="" fill="none" stroke="#3f5787" stroke-width="0.5" />
+          <path
+            id="top"
+            d="M12.473017,77.858673c0,0,12.412281,2.607693,21.010105-4.357633C50.800657,59.471645,52.624768,12.449248,96.947764,12.449248s44.968303,41.17425,59.988902,56.761663c9.324112,9.675965,26.158727,8.647762,26.158727,8.647762"
+            transform="translate(.000002 0.000001)"
+            fill="none"
+            stroke="#059865"
+            stroke-width="25"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            stroke-dashoffset="522.24"
+            stroke-dasharray="261.118752"
+          />
+          <path
+            id="bottom"
+            d="M12.473018,114.104h170.622376"
+            transform="translate(.000001-.213255)"
+            fill="none"
+            stroke="#056598"
+            stroke-width="25"
+            stroke-linecap="round"
+            stroke-dashoffset="391"
+            stroke-dasharray="195.498402"
+          />
+        </svg>
+      </div>
+    </div>
+    <style>
```

Comment:
Usually I would agree, but in this case specifically, we want the loader to be displayed a fast a possible and splitting this CSS and JS in other files will delay the display

Reply:
I understand so importing it is not an option as it will cause the page to "flash" with the unstyled content before the CSS is loaded ? 

But <style> is not allowed in the <body> tag.

According to [this post](https://stackoverflow.com/questions/49523627/i-want-to-load-specific-stylesheet-first-in-html-before-other-sylesheets-and-bod), the best solution is to hardcode the CSS in <head> : it's W3C compliant + it's loaded before the <body>.

Reply:
Whoa! I didn't know that the style tag in body what such a controversial topic, and that the _current_ spec does not allow it ...
https://stackoverflow.com/questions/2830296/using-style-tags-in-the-body-with-other-html#answer-50154225
https://github.com/whatwg/html/issues/1605

Ok let move the CSS into the head